signalfx-org-metrics/docs/sf.org.numDimensionNamesCreated.md

```diff
@@ -0,0 +1,10 @@
+---
+title: sf.org.numDimensionNamesCreated
+brief: Number of new dimension names that were created
+metric_type: gauge
+---
+### sf.org.numDimensionNamesCreated
+
+Count of new dimension names that were created in the past 5 minutes in this organization
```

Comment:
Use "Number of" instead of "Count of" throughout; using the term Count kind of conflicts with metric type being gauge.